docs/content/guides/author-apps/kubernetes/how-to-patch-pod/index.md

```diff
@@ -0,0 +1,133 @@
+---
+type: docs
+title: "How-To: Patch Kubernetes resources using PodSpec"
+linkTitle: "Patch using PodSpec"
+description: "Learn how to patch Kubernetes resources using PodSpec definitions"
+weight: 300
+slug: 'patch-podspec'
+categories: "How-To"
+tags: ["containers","Kubernetes"]
+---
+
+This how-to guide will provide an overview of how to:
+
+- Patch Radius-created Kubernetes resources using [PodSpec](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec) definitions
+
+## Prerequisites
+
+- [rad CLI]({{< ref getting-started >}})
+- [Radius initialized with `rad init`]({{< ref howto-environment >}})
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+
+## Step 1: Define an app and a container
+
+Begin by creating a file named `app.bicep` with a Radius application and [container]({{< ref "guides/author-apps/containers" >}}):
+
+{{< rad file="snippets/patch-container.bicep" embed=true >}}
+
+## Step 2: Deploy the app and container
+
+1. Deploy and run your app:
+
+   ```bash
+   rad run ./app.bicep
```

Comment:
Without an explicit "Run rad init in a new directory and specify 'y' we'll need to pass in the application at run:
```suggestion
   rad run ./app.bicep -a demo
```